packages/common/src/types/NodeInfo.ts

```diff
@@ -1,6 +1,4 @@
-import { BakerId, Duration, Timestamp } from '../types';
-
-export type PeerId = HexString;
+import { BakerId, Duration, HexString, Timestamp } from '../types';
```

Comment:
Wait how did you reference HexString before without importing it :thinking:

Reply:
You suggested changing it to a `HexString` and I commited the suggestion in commit `90208bf`, but the suggestion didn't include an import statement. Of course this meant that it could not build too.